src/plugins/index.js

```diff
@@ -36,6 +37,42 @@ export default class Plugins {
     this.lock = new Lock(this.out)
   }
 
+  clearCache (...paths: string[]) {
+    this.cache.deletePlugin(...paths)
+  }
+
+  async _installCheck (name: string, tag: string) {
+    // will be overridden in Plugins
+  }
+
+  async install (name: string, tag: string = 'latest') {
```

Comment:
@dickeyxxx the reason that this is in `plugins.js` rather than `user.js` is due to the refactor where I moved all the caching logic up into `plugins.js` to simplify the `user.js` code.  I am fine with pushing the cache back down into `manager.js` if you think having this in `user.js` would be a better separation.